problemsets-mat223/linearalbegra.tex

```diff
@@ -4491,16 +4491,73 @@ \section*{Inverses}
 	\]
 	\begin{parts}
 		\item What is $A^{-1}A$?
+			\begin{solution}
+				$A^{-1}A = I$. This is true by definition of an inverse, but we
```

Comment:
Pick one of "this is true by definition" or "this is true by *the* definition of an inverse"